### PR TITLE
Fix stream type in mailparser-mit

### PR DESCRIPTION
**Pull request by barspielberg**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] ~~If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.~~

https://github.com/mazira/mailparser-mit/blob/master/lib/mailparser.js#L70

### CODE REVIEW

File: types/mailparser-mit/index.d.ts [hunk 1]
```diff
@@ -90,8 +95,14 @@ export interface ParsedEmail {
     attachments?: Attachment[];
 }
 
-export class MailParser extends Stream {
+export class MailParser extends Stream implements NodeJS.WritableStream {
     constructor(options?: MailParserOptions);
+    writable: boolean;
```
Comment (atlowChemi) [inline types/mailparser-mit/index.d.ts:100]: should this be?
```suggestion
    writable: true;
```

**Review by atlowChemi — APPROVED** [review]

**Comment by barspielberg** [comment]
Ready to merge

**Review by atlowChemi — COMMENTED** [review]
Thank you for taking the time and fixing these types 🙂

**Review by atlowChemi — APPROVED** [review]

**Comment by barspielberg** [comment]
Ready to merge

**Pull request merged**